pdns/.gitignore

```diff
@@ -51,3 +51,4 @@ version_generated.h
 /calidns
 /dumresp
 /htmlfiles.h
+/dnsdistdist
```

Comment:
This line would mean we'd have to `git add --force` new files in that directory, which is a recipe for disaster (or, at least *problems*)

It *might* be a better idea to just run `./populate`, keep the symlinks in the repository and remove the `populate` script. This way we can ignore all generated files with the `dnsdistdist/.gitignore`